src/databricks/labs/ucx/mixins/sql.py

```diff
@@ -264,10 +265,17 @@ def _row_converters(self, execute_response):
             type_name = col.type_name
             if not type_name:
                 type_name = ColumnInfoTypeName.NULL
-            conv = self.type_converters.get(type_name, None)
+            if type_name == ColumnInfoTypeName.BOOLEAN:
+                conv = self._get_convert_boolean_type_function
+            else:
+                conv = self.type_converters.get(type_name, None)
             if conv is None:
                 msg = f"{col.name} has no {type_name.value} converter"
                 raise ValueError(msg)
             col_conv.append(conv)
         row_factory = type("Row", (Row,), {"__columns__": col_names})
         return col_conv, row_factory
+
+    @staticmethod
+    def _get_convert_boolean_type_function(value):
+        return literal_eval(value.capitalize())
```

Comment:
```suggestion
        return value.lower() == 'true'
```
or something like that. don't pull it unnecessary libraries